### PR TITLE
fix(CDPUserDashboard): pretty date rendering for merged surveys

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
This PR improves the overall display of survey results

### CODE REVIEW

**Review by philip-cline — APPROVED** [review]
Splendid

File: util/ui.ts [hunk 1]
```diff
@@ -97,10 +97,12 @@ export const getActiveUserTypes = (): {
...
-export const getDateFromCDPFileName = (filename: string): string => {
-  const TRIP_DATA_SUFFIX = '-anon-trip-data'
-  const date = filename.split(TRIP_DATA_SUFFIX)?.[0].split('/')?.[1]
-  if (!date || !filename.includes(TRIP_DATA_SUFFIX)) return filename
+export const getDateFromCDPFileName = (
+  filename: string,
+  tripDataSuffix: string
+): string => {
+  const date = filename.split(tripDataSuffix)?.[0].split('/')?.[1]
+  if (!date || !filename.includes(tripDataSuffix)) return filename
```
Comment (binh-dam-ibigroup) [inline util/ui.ts:105]:
```suggestion
  if (!date || !filename.endsWith(tripDataSuffix)) return filename
```

**Review by binh-dam-ibigroup — APPROVED** [review]
Definitely easier to read, with one minor suggestion.

File: util/ui.ts (hunk 1 above)
Comment (miles-grant-ibigroup) [reply util/ui.ts:105]: I think the string technically ends with `.zip`!

**Pull request merged**